client/css/widgets/basicwidget.css

```diff
@@ -9,3 +9,7 @@ body.edit .widget.basic.hasImage {
   opacity: 0.45;
 }
 
+.shadow {
+  filter: brightness(0);
+  opacity: 0.3;
+}
```

Comment:
I think `pointer-events: none` would be useful. Also I think the class name should be a bit more specific; perhaps `dragging-shadow`

Reply:
Sounds good, done.